webrender/src/renderer.rs

```diff
@@ -2843,8 +2843,18 @@ impl Renderer {
 
             for alpha_batch_container in &target.alpha_batch_containers {
                 if let Some(target_rect) = alpha_batch_container.target_rect {
+                    // Note: `framebuffer_target_rect` needs a Y-flip before going to GL
+                    let rect = if render_target.is_none() {
```

Comment:
Probably worth putting this code in a function instead of duplicating below?

Reply:
I think this should be done to serve both these scissors and ones for clearing. It's not even going to save LOC, just give that piece of logic a name.